src/components/MultiHopTrade/components/SlippagePopover.tsx

```diff
@@ -122,16 +123,18 @@ export const SlippagePopover: FC<SlippagePopoverProps> = memo(
 
     return (
       <Popover isLazy placement='bottom-end' onClose={handleClose}>
-        <PopoverTrigger>
-          <Tooltip isDisabled={!isDisabled} label={translate(tooltipTranslation)}>
-            <IconButton
-              aria-label={translate('trade.tradeSettings')}
-              icon={faGear}
-              variant='ghost'
-              isDisabled={isDisabled}
-            />
-          </Tooltip>
-        </PopoverTrigger>
+        <Tooltip isDisabled={!isDisabled} label={translate(tooltipTranslation)}>
+          <Box display='inline-block'>
```

Comment:
The magic bit - this simply wraps `<PopoverTrigger />` in a `<Box />`, and moves the `<Tooltip />` wrapping outside of the box/tooltip node.

View without whitespaces diff for the actual diff

<img width="379" alt="image" src="https://github.com/shapeshift/web/assets/17035424/c26376c8-32f3-4dce-9316-922f796e2835">